app-directory/css-in-js/package.json

```diff
@@ -9,23 +10,24 @@
     "lint": "next lint"
   },
   "dependencies": {
-    "@chakra-ui/react": "^2.4.6",
-    "@emotion/react": "^11.10.5",
-    "@emotion/styled": "^11.10.5",
+    "@chakra-ui/react": "^2.5.1",
+    "@emotion/react": "^11.10.6",
+    "@emotion/styled": "^11.10.6",
     "@vercel/examples-ui": "latest",
     "framer-motion": "^6.5.1",
     "next": "latest",
     "react": "latest",
     "react-dom": "latest"
   },
   "devDependencies": {
-    "@types/node": "^17",
+    "@types/node": "^17.0.45",
     "@types/react": "latest",
-    "autoprefixer": "^10.4.7",
+    "autoprefixer": "^10.4.14",
     "eslint": "latest",
     "eslint-config-next": "canary",
-    "postcss": "^8.4.14",
-    "tailwindcss": "^3.0.24",
-    "typescript": "^4.7.3"
+    "postcss": "^8.4.21",
+    "tailwindcss": "^3.2.7",
+    "turbo": "^1.8.3",
```

Comment:
Do we have renovate/dependabot on these? Or should we use `latest` on this? cc @gsoltis on whether this repo should be part of our testing scenarios for releases